modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -69,8 +69,8 @@ kind: NUMAResourcesScheduler
 metadata:
   name: numaresourcesscheduler
 spec:
-  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-container-rhel8:v{product-version}"
-  cacheResyncPeriod: "5s" <1> 
+  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-container-rhel9:v{product-version}.1-2"
```

Comment:
let's not specify the specific tag. the version tag (X.Y) is expected to pull latest image on that version